src/components/LanguageSelection/LanguageSelection.js

```diff
@@ -21,11 +21,17 @@ const DEFAULT_HEADING = (
 );
 
 
+const changeLanguagePrompt = defineMessage({
+  description: 'change language prompt',
+  defaultMessage: 'Changing language will empty the form. Continue?'
```

Comment:
I'd use `Changing the language will require you to restart the form. Are you sure you want to continue?`